tests/integrations/test_crossed_integrations.py

```diff
@@ -64,17 +67,18 @@ def test_produce(self):
         # assert self.consume_response.status_code == 200
 
         # The weblog is the producer, the buddy is the consumer
-        # The buddy is the producer, the weblog is the consumer
         self.validate_kafka_spans(
             producer_interface=interfaces.library,
             consumer_interface=interfaces.python_buddy,
             topic=self.WEBLOG_TO_BUDDY_TOPIC,
         )
 
     @missing_feature(library="python")
+    @missing_feature(library="java")
     def test_produce_trace_equality(self):
+        """This test relies on the setup for produce, it currently cannot be run on its own"""
         producer_span = self.get_span(interfaces.library, span_kind="producer", topic=self.WEBLOG_TO_BUDDY_TOPIC)
-        consumer_span = self.get_span(interfaces.python_buddy, span_kind="consumer", topic=self.BUDDY_TO_WEBLOG_TOPIC)
+        consumer_span = self.get_span(interfaces.python_buddy, span_kind="consumer", topic=self.WEBLOG_TO_BUDDY_TOPIC)
```

Comment:
I'm surprised, we messed up this line in the first place ?

Reply:
well, yes